.travis.yml

```diff
@@ -77,19 +77,19 @@ jobs:
       script:
         - unbuffer node build-system/pr-check/remote-tests.js
       after_script:
-        - build-system/sauce_connect/stop_sauce_connect.sh
+        - ps -ef
     - stage: test
       name: "End to End Tests"
       script:
         - unbuffer node build-system/pr-check/e2e-tests.js
+      after_script:
+        - ps -ef
 cache:
   directories:
     - node_modules
     - build-system/tasks/e2e/node_modules
     - build-system/tasks/visual-diff/node_modules
-    - sauce_connect
```

Comment:
This is the directory to which `build-system/sauce_connect/start_sauce_connect.sh` downloads the proxy. Caching it [avoids](https://travis-ci.org/ampproject/amphtml/jobs/547876254#L1980) a fresh download for each build.